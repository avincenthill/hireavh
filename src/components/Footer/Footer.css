.footer-container {
  color: var(--c-1);
  background-color: var(--c-black);
  position: fixed;
  bottom: 0;
  width: 100%;
  height: var(--s-s);
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: var(--fs-xxs);
  z-index: var(--z-top);
  border-top: var(--b-1) var(--c-1) solid;
}