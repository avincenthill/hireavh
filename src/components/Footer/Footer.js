import React from "react";
import content from "../../content/content";
import Nav from "../Nav/Nav";
import "./Footer.css";

function Footer() {
  return (
    <div>
      <Nav isBottom={true}></Nav>
      <div className="footer-container">
        <span>{content.footer.copywrite}</span>
      </div>
    </div>
  );
}

export default Footer;
