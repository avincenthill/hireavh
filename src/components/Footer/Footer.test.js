import Footer from "./Footer";
import React from "react";
// import { shallow } from "enzyme";
import renderer from "react-test-renderer";

describe("Footer", () => {
  describe("snapshot", () => {
    it("should match the last saved snapshot", () => {
      const tree = renderer.create(<Footer />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
