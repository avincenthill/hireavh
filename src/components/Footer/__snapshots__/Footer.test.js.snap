// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot should match the last saved snapshot 1`] = `
<div>
  <div
    className="nav nav-bottom"
  >
    <div
      className="nav-icon $
          this.props.isBottom ? \\"nav-icon-bottom\\" : \\"nav-icon-top\\"
        }"
    >
      <svg
        fill="currentColor"
        height="1em"
        onClick={[Function]}
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": "var(--c-1)",
            "height": "3rem",
            "width": "3rem",
          }
        }
        viewBox="0 0 8 16"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M8 4v1H0V4h8zM0 8h8V7H0v1zm0 3h8v-1H0v1z"
          fillRule="evenodd"
        />
      </svg>
    </div>
  </div>
  <a
    className="footer-container"
    href="/about"
  >
    <span>
      copyright Alex Vincent-Hill 2020 - üè† about
    </span>
  </a>
</div>
`;
